gnowsys-ndf/gnowsys_ndf/ndf/templates/ndf/lms.html

```diff
@@ -81,53 +76,80 @@
+      {% if not group_object.project_config.tab_name %}{% trans "Lessons" %}{% else %}{{group_object.project_config.tab_name}} {% endif %}</a>
+      {% if is_gstaff %}<a href="{% url 'unit_detail' groupid %}" title="Edit Structure" style="background-color:#6f0859; border-radius:4px; margin-left:-13px; width:0px;"><i class="fa fa-pencil" style="margin-left:-7px; color:white;"></i>{% endif %}</a>
+     </li>
+    {% if 'Group' not in group_object_member_of_names_list  or  'base_unit'  in group_object_member_of_names_list  %}
+    <li>  <a href="{% url 'course_raw_material' group_id %}" {% if title == 'raw material' or title == "raw_material_detail"  %}class="selected"{% endif %} tabindex="3" title="Contains useful files such as images, audios, videos etc."><i class="fi-folder "></i> {% trans "Resources" %}</a> </li>
+    {% endif %}
+    <li>
+    <li>  <a  href="{% url 'course_gallery' group_id %}" {% if title == 'gallery' or title == "asset_gallery_detail" %} class="selected" {% endif %} tabindex="4" title="Upload and share your files such as images, audios, videos, spreadsheets"> <i class="fi-photo "></i>{% if 'Group'  in group_object_member_of_names_list and 'base_unit' not in group_object_member_of_names_list %} {% trans "E-Library" %} {% else %} {% trans "Gallery" %}{% endif %}</a> </li>
+    <li>
+      <a href="{% url 'course_notebook' group_id %}" {% if title == 'notebook' %}class="selected"{% endif %} tabindex="5" title="Create and share notes, read notes shared by all students"><i class="fi-book "></i>
+      {% if 'Group' in group_object_member_of_names_list and  'base_unit' not in group_object_member_of_names_list %}
+        {% trans "Blog" %}
+      {% else %}
+        {% trans "e-Notes" %}
+      {% endif %}
+      </a>
+    </li>
+  </li>
+    {% endcomment %}
 </ul>
-<ul class="nav_menu_1 authoring-tab">
-  {% if is_gstaff %}
-    {% if request.author.agency_type != 'Teacher' or group_object.agency_type == 'School' or "Author" in group_object_member_of_names_list %}
+<ul class="nav_menu_1 authoring-tab" style="width:50%; display:inline">
+
+    {% if is_gstaff and request.author.agency_type != 'Teacher' or group_object.agency_type == 'School' %}
     <li>
         <a href="{% url 'course_pages' group_name %}"  {% if title == 'course_pages' or title == 'create_course_pages' %}class="selected"{% endif %}><i class="fi-page-filled "></i> {% trans "Activities" %}</a>
     </li>
     <li>
         <a href="{% url 'asset_list' group_name %}" {% if title == 'asset_list' or title == 'asset_detail' or title == 'asset_content_detail'%}class="selected"{% endif %}><i class="fi-folder-lock"></i> {%trans "Assets" %}</a>
     </li>
-    {% if group_object.agency_type != 'School' and "Author" not in group_object_member_of_names_list  %}
+    {% if group_object.agency_type != 'School' %}
```

Comment:
While making changes in any of the logical conditions, test it with following use users:
1. Anonymous
2. Normal 
3. Group Admin
4. Super